sdk/web-pubsub/web-pubsub-express/package.json

```diff
@@ -28,16 +25,14 @@
     "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "echo \"Browser is not supported.\" && exit 0",
-    "unit-test:node": "dev-tool run test:node-ts-input --no-test-proxy",
+    "unit-test:node": "dev-tool run test:vitest --no-test-proxy",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "files": [
     "dist/",
-    "dist-esm/",
-    "types/web-pubsub-express.d.ts",
-    "types/web-pubsub-express.d.ts.map",
     "README.md",
-    "LICENSE"
+    "LICENSE",
+    "dist-esm/src"
```

Comment:
why keeping `dist-esm/src`? Are there other files generated into it?

Reply:
No, it was a mistake.  Fixed.